okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -136,6 +140,17 @@ public Level getLevel() {
     return level;
   }
 
+  /** Change the limitation of response and request body logs size. */
+  public HttpLoggingInterceptor setBodyLogMax(long bodyLogMax) {
```

Comment:
I don't think "Log" needs to be in the method name here. Maybe `setMaxBodyLength`?